filament/backend/include/backend/ShaderStageFlags.h

```diff
@@ -35,6 +35,10 @@ struct ShaderStageFlags {
 
 constexpr ShaderStageFlags ALL_SHADER_STAGE_FLAGS = { .vertex = true, .fragment = true };
 
+inline bool test(ShaderStageFlags flags, ShaderType type) {
```

Comment:
this needs a better name

Reply:
In Filament, struct can have a member function?

Reply:
I just wanted something more descriptive that `test` -- maybe `hasShaderType()`. And why not make it a method of `ShaderStageFlags`, this way you could write: `stages.hasShaderType(type)`.

Reply:
Thanks I will add `hasShaderType()`. FYI `test` comes from `std::bitset<N>::test`.